packages/core/core/src/factories/apiClientFactory/index.ts

```diff
@@ -11,39 +11,38 @@ import { applyContextToApi } from './context';
 const isFn = (x) => typeof x === 'function';
 
```

Comment:
about this file:
- Please keep before/after create as it is (not async) - I know that is the opposite as it was in the AC but it doesn't make sense to have before/after configure async. In that way, createApiClient won't be async as well.
- beforeCall/After call just prefix async is enough:

```    const extensionHooks = {
      before: async (params) => lifecycles
        .filter(e => isFn(e.beforeCall))
        .reduce((args, e) => e.beforeCall({ ...params, configuration: settings.config, args}), params.args),
      after: async (params) => lifecycles
        .filter(e => isFn(e.afterCall))
        .reduce((response, e) => e.afterCall({ ...params, configuration: settings.config, response }), params.response)
    };
```